test/e2e/scheduling/preemption.go

```diff
@@ -94,7 +95,9 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 		var err error
 		for _, pair := range priorityPairs {
 			_, err := f.ClientSet.SchedulingV1().PriorityClasses().Create(context.TODO(), &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: pair.name}, Value: pair.value}, metav1.CreateOptions{})
-			framework.ExpectEqual(err == nil || apierrors.IsAlreadyExists(err), true)
+			if !(err == nil || apierrors.IsAlreadyExists(err)) {
```

Comment:
You can simplify the expression.

```suggestion
			if err != nil && !apierrors.IsAlreadyExists(err) {
```

Reply:
Done!